source/slang/slang-emit-c-like.cpp

```diff
@@ -3450,6 +3466,11 @@ bool CLikeSourceEmitter::isTargetIntrinsic(IRInst* inst)
     return findTargetIntrinsicDefinition(inst, intrinsicDef);
 }
 
+bool CLikeSourceEmitter::isEmmitingHeader() {
```

Comment:
Did you mean `isEmittingHeader()'?

Reply:
I think the name will be better with "shouldXXX()" and probably something like "shouldEmitOnlyHeader()".
Because the compiler option is a global state that isn't gonna change once set.
"isEmitting" sounds like a transitional state where it can be in a emitting state only temporarily.

Also note that the other compiler options are queried with functions whose name starts with "should".
```
        bool shouldEmitSPIRVDirectly()
        {
            return getBoolOption(CompilerOptionName::EmitSpirvDirectly);
        }
        bool shouldUseScalarLayout()
        {
            return getBoolOption(CompilerOptionName::GLSLForceScalarLayout);
        }
```